scripts/oneoff/inject-framework-dates.py

```diff
@@ -26,20 +26,23 @@
         'applicationsCloseAtUTC': '2016-01-19T15:00:00Z',
         'intentionToAwardAtUTC': '2016-02-18T12:00:00Z',
         'frameworkLiveAtUTC': '2016-03-03T12:00:00Z',
+        'frameworkExpiresAtUTC': '1970-01-01T00:00:00Z',
```

Comment:
Should `'frameworkExpiresAtUTC'` for DOS1 be `'2017-01-27T12:00:00Z'`? We turned DOS1 off and DOS2 on at the same time. I know it's not in the dates yml file but neither is live date.

Reply:
Sounds good to me.